github-page/_guides/quick-start-guide.md

```diff
@@ -110,15 +110,19 @@ class KittenClass {
 
 const Kitten = getModelForClass(KittenClass);
 
-let document = await Kitten.create({ name: 'Kitty' });
+let document = new Kitten({ name: 'Kitty' });
 // "document" has types of KittenClass
 ```
 
-Please note that `new Kitten({})` or `await Kitten.create({})` has no types of KittenClass, because typegoose dosnt modify functions of mongoose
+Please note that `await Kitten.create({})` has no types of KittenClass, because typegoose doesn't modify functions of mongoose
```

Comment:
```suggestion
Please note that `new Kitten({})` & `await Kitten.create({})`'s `{}` has no types of KittenClass, because typegoose doesn't modify functions of mongoose
```